tensor2tensor/trax/layers/initializers.py

```diff
@@ -30,21 +42,70 @@ def Init(shape, rng):
   return Init
 
 
-def GlorotNormalInitializer(out_dim=0, in_dim=1, scale=onp.sqrt(2)):
-  """An initializer function for random Glorot-scaled coefficients."""
+def RandomUniformInitializer(lim=1.0):
+  """An initializer function for random uniform coefficients."""
   def Init(shape, rng):
-    fan_in, fan_out = shape[in_dim], shape[out_dim]
-    size = onp.prod(onp.delete(shape, [in_dim, out_dim]))
-    std = scale / backend.numpy.sqrt((fan_in + fan_out) / 2. * size)
-    return (std * backend.random.normal(rng, shape)).astype('float32')
+    return (backend.random.uniform(rng, shape, -lim, lim)).astype('float32')
   return Init
 
 
-def GlorotUniformInitializer(out_dim=0, in_dim=1):
-  """An initializer function for random uniform Glorot-scaled coefficients."""
-  def Init(shape, rng):
-    fan_in, fan_out = shape[in_dim], shape[out_dim]
-    std = backend.numpy.sqrt(2.0 / (fan_in + fan_out))
-    a = backend.numpy.sqrt(3.0) * std
-    return backend.random.uniform(rng, shape, minval=-a, maxval=a)
-  return Init
+def VarianceScalingInitializer(scale, mode, distribution):
+    """Initializer capable of adapting its scale to the shape of weights tensors."""
+    if scale <= 0.:
+        raise ValueError(f"scale must be positive float, {scale} given")
```

Comment:
We need both py2 and py3 comatibility, so please don't use f"aaa" -- just do 'aaa' instead. (And note that in trax we try to use single-quotes.)